Tests/TestAppUITests/TestAppUITests.swift

```diff
@@ -23,6 +28,24 @@ class TestAppUITests: XCTestCase {
             XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
         }
     }
+
+    func testScreenInitThrowsWhenScreenWithMultipleElementsIsNotLoaded() throws {
+        do {
+            _ = try MissingMultipleElementsScreen(app: app)
+            XCTFail("Expected `ScreenObject` `init` to throw, but it didn't")
+        } catch {
+            XCTAssertEqual(error as? ScreenObject.WaitForScreenError, .timedOut)
+        }
```

Comment:
Swift functions can "throw" errors. This is a cool mechanism to ensure errors are handled properly by the developers using that code, because the compiler won't compile the code unless the error is handled. There are different ways to handle thrown errors, but the most common one, _I think_, is to use a "do catch" construct. 

Basically, you wrap the code that might throw (which the compiler forces you to call with `try`) in a `do { }` block. This block _has to_ be followed by a `catch { }` one where you can access the caught `error` and, ideally, handle it.

In this test, I'm using the "do catch" to make sure trying to initialize a screen that expects elements that are not loaded throws an error. In the `catch` block, I also check that the error is the one I expect it to be. This last part is useful to prevent accidental changes to the type of error thrown.

I use the word "accidental" because, to me, every change to the codebase that is not reflected in the tests first is not intentional. That's the Test-Driven Developer in me talking 😅